Amplify/DevMenu/Data/PluginInfoHelper.swift

```diff
@@ -14,86 +14,70 @@ struct PluginInfoHelper {
     static func getPluginInformation() -> [PluginInfoItem] {
         var pluginList = [PluginInfoItem]()
 
-        // Add Analytics plugins information
```

Comment:
Does this work?
```swift
let plugins = Amplify.Analytics.plugins 
                     + Amplify.API.plugins 
                     + Amplify.Auth.plugins
                     + Amplify.DataStore.plugins ...
pluginList.append(contentsOf: plugins.map {
                 makePluginInfoItem(for: $0.key, versionable: $0.value as? AmplifyVersionable)
             })
```

Reply:
No, because `plugins` is an `internal` implementation detail of the category. We're diving into it here because we happen to be in the same package, but you'll note special case handling for `API` and `Logging` categories. If we wanted to abstract it out, we could write a `allPlugins` property that would normalize that, but it's not something we have a public facing use case for just now, so the logic would be manually attached to each category. Since we're only doing it for this dev menu, I'm disinclined to add API surface over on each category separated from the call site. (The very reasonable counterargument of course is that if a future internal use case needs something like an `allPlugins` property, it will be blind luck for someone to remember "Oh yeah, the Dev Menu is doing special case handling for this, I should change it." But I'm going to solve for the current problem rather than adding complexity for a hypothetical future case.)